app/components/youtube-videos.js

```diff
@@ -0,0 +1,36 @@
+import { inject as service } from '@ember/service';
```

Comment:
Maybe place this file within the 'index' directory in the components. Since it is related to the index page only